interfaz_controller/interfaz_controller.py

```diff
@@ -1,21 +1,18 @@
+import numpy as np
 import xlwings as xw
 
 
-def main():
+def main(workbook: xw.Book = None):
     wb = xw.Book.caller()
     sheet = wb.sheets[0]
-    if sheet["A1"].value == "Hello xlwings!":
-        sheet["A1"].value = "Bye xlwings!"
-    else:
-        sheet["A1"].value = "Hello xlwings!"
 
-
-@xw.func
-def hello(name):
-    return f"Hello {name}!"
+    # Cells B4 to B8 are called with the name Ranges, defined on the name manager from Ms Excel
+    params = sheet['Ranges'].options(np.array, transpose=True).value
+    sheet['B10'].value = 7758 * params[0] * params[1] * params[2] * (1 - params[3]) / params[4]
```

Comment:
The cell that displays the value of STOIIP should also have a named range. When I add a row in excel, the result now calculates correctly but in another cell. Correct this please

Reply:
Done